marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/step.sjs

```diff
@@ -116,7 +116,7 @@ class Step {
     try {
       stepModule = this.hubUtils.retrieveModuleLibrary(moduleUri);
     } catch (e) {
-      httpUtils.throwBadRequest(`Unable to access module: ${moduleUri}. Verify that this module is in your modules database and that your user account has a role that grants read permission to this module.`);
+      httpUtils.throwBadRequest(`Unable to access module: ${moduleUri}. Verify that this module is in your modules database and that your user account has a role that grants read and execute permission to this module`);
```

Comment:
@srinathgit  Can you test one more thing - see what happens if the step module is present but has a compilation error in it. I think we may get the same error, in which case we really ought to include "e.stack" at the end of the message - e.g. "; cause: ${e.stack}".

Reply:
We cannot have the same message if there is compilation error right ? I have changed the message to 
```
`Cannot run module: ${moduleUri}; cause: ${e.stack}`
```
Let me know if that's ok.